Proj0045: Convention-based MSBuild file names should use correct casing (#119)
dotnet-project-file-analyzers/dotnet-project-file-analyzers.github.io: rules/Proj0045.md

---
parent: General
ancestor: MSBuild
---

# Proj0045: Convention-based MSBuild file names should use correct casing

> Some MSBuild files have a certain role during the build based on a name

Author: naming
```suggestion
Some MSBuild files have a certain role during the build based on a naming
```

> convention. Although MSBuild is mostely case-insensitve this having different

Author: ```suggestion
convention. Although MSBuild is mostely case-insensitive this having different
```

> convention. Although MSBuild is mostely case-insensitive this having different
> casing might cause issues, specially on files systems (Unix for instance) that

Reviewer: ```suggestion
convention. Although MSBuild is mostly case-insensitive, this having different
casing might cause issues, especially on files systems (Unix for instance) that
```

are case-sensitve. This rule will report when the casing of such convention
based files is off.

## Compliant

> Files for which the casing is checked.

Reviewer: ```suggestion
Files for which the casing is checked:
```
